### PR TITLE
feature: add new $refresh helper

### PR DESCRIPTION
**Pull request by ryangjchandler**
This pull request adds a new `$refresh` helper. It can be used to manually refresh a component, better than forcefully changing a piece of data.

Under the hood, it just calls `__x.updateElements`, but `$refresh()` is much nicer (of course).

### CODE REVIEW

**Review by SimoTod — COMMENTED** [review]
Looks good. We need to add an example in refresh.html (empty body at the minute)

**Comment by ryangjchandler** [comment]
> Looks good. We need to add an example in refresh.html (empty body at the minute)

Ha, it's because I cut the content instead of copying it, lol.

**Review by SimoTod — APPROVED** [review]

**Comment by ryangjchandler** [comment]
I'll wait for @KevinBatdorf to take a look and approve before merging this in, it's his baby after all - I know he's a very busy man! 🤙🏼

**Review by HugoDF — APPROVED** [review]

File: src/refresh.js
Comment (KevinBatdorf) [inline src/refresh.js:12]: I think we can do:

```js
return (el = $el) => el.__x.updateElements(el)
```

and get it working with the `$component` helper after too after #76

**Review by KevinBatdorf — CHANGES_REQUESTED** [review]
Good idea! I think we can let the user pass in a $component with a minor change. 

Also, maybe there's a benefit in keeping the helpers in alphabetical order?

I can clean all of that up though. We should probably add a codepen too

File: src/refresh.js
Comment (ryangjchandler) [reply src/refresh.js:12]: Ah, good shout. Hadn't though about that!

**Comment by KevinBatdorf** [comment]
This was awesome. Let's release it tomorrow (Friday)

I added the param to refresh other components and just moved things around to keep them in alphabetical order (might as well!). Will merge now

**Pull request merged**
